core/src/components/fab/test/basic/fab.e2e.ts

```diff
@@ -0,0 +1,67 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('fab: basic', () => {
```

Comment:
Is there a reason we repeat:
```
await page.goto(`/src/components/fab/test/basic`);
```

vs. adding it into a `beforeEach`?

Reply:
Some of the tests have skips in them, and I figured calling `goto` explicitly each time allowed for a perf improvement where the skips can short-circuit the navigation.

On reassessment though, I can just split the tests up into different `describe`s and clean things up that way 👀